### ISSUE TITLE
Related resources asking update to 1.0.4 and can’t dismiss over web ..

### ISSUE THREAD

**Issue by Bouli911** [nextcloud/related_resources#175]
![B60781BC-2B2A-465D-B714-84C1BDF0FEE4](https://user-images.githubusercontent.com/116047215/213874867-4a18c00d-bc25-47de-9fe6-c465ec6b7946.jpeg)
![36929051-0BCA-4C81-AA32-CBBF767C052B](https://user-images.githubusercontent.com/116047215/213874875-1fbb1b35-104a-47f4-bd92-a7d180dee1de.jpeg)

Solution to solve problems :
I remove notifications thought client nextcloud app IOS . 
Unable to do it thought web ..

Nextcloud 25.0.3
Php Version: 8.1.14
mysql Version: 10.6.11
Operating System: Linux 5.15.0-58-generic x86_64

**Comment by jniggemann** [comment]
I have the same problem on 25.0.3 with php 8.2.1 and MariaDB !0.6.11.

On the NC forums there are also people having this issue:
https://help.nextcloud.com/t/app-related-resources-cannot-be-updated/154177

**Comment by mbentley** [comment]
It seems like there is definitely a issue in how the logic works for app updates. This is the 2nd time I've seen where an app needs to pull an update due to an issue but the notifications continue to spam the user about an update that is no longer available.

**Comment by Foxeronie** [comment]
Sorry for pinging you @ArtificialOwl , but after one month of daily popups to update this app without any updates available maybe you could have a look at this please?

Thanks in advance.

**Comment by mbentley** [comment]
I doubt there is anything that can be done directly by an app.  Looking in the database, I was able to delete the rows related to `releated_resources` if you're comfortable with direct db manipulation.  It's in the `oc_notifications` table.  Should be obvious as I believe it specifically had `related _resources` in the `object_type`.

```
SELECT * FROM `oc_notifications` WHERE `subject` = "update_available";
```

No guarantees that deleting these rows won't cause problems but I no longer receive notifications about related resources and other new notifications work fine as expected.

**Comment by Bouli911** [comment]
It seems to be fixed in last release of Nextcloud 25.0.4 .

Now I can dismiss thought web client .

**Comment by szaimen** [comment]
Should be fixed with 25.0.4 and 26
